### PR TITLE
70pre opensearch py rewrite

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## 🗒️ Summary
Supporting work to facilitate work on #70
Replaces raw-request OpenSearch interaction with `opensearch-py` package 

## ⚙️ Test Data and/or Report
Tests pass.  Manually tested as a sanity check for db reads/writes against I&T docker registry

## ♻️ Related Issues
related to #70

### CODE REVIEW

File: src/pds/registrysweepers/utils/db/__init__.py [hunk 1]
```diff
@@ -22,59 +24,59 @@
...
     more_data_exists = True
+    scroll_id = None
     while more_data_exists:
-        resp = retry_call(
-            auto_raise_for_status(requests.get),
-            fargs=[urllib.parse.urljoin(host.url, path)],
-            fkwargs={"auth": (host.username, host.password), "verify": host.verify, "json": req_content},
-            exceptions=(HTTPError, RuntimeError),
+        if scroll_id is None:
+            fetch_func = lambda: client.search(  # noqa: E731
```
Comment (nutjob4life) [inline src/pds/registrysweepers/utils/db/__init__.py:54]: See Slack discussion for assigning lambdas to variables

File: src/pds/registrysweepers/utils/db/__init__.py [hunk 2]
```diff
@@ -22,59 +24,59 @@
...
+                index=index_name,
+                body=query,
+                scroll=scroll_keepalive,
+                size=page_size,
+                _source_includes=_source.get("includes", []),  # TODO: Break out from the enclosing _source object
+                _source_excludes=_source.get("excludes", []),  # TODO: Break out from the enclosing _source object
+            )
+        else:
+            fetch_func = lambda: client.scroll(scroll_id=scroll_id, scroll=scroll_keepalive)  # noqa: E731, B023
```
Comment (nutjob4life) [inline src/pds/registrysweepers/utils/db/__init__.py:63]: ʺ

**Review by nutjob4life — CHANGES_REQUESTED** [review]

File: src/pds/registrysweepers/utils/db/__init__.py (hunk 2 above)
Comment (alexdunnjpl) [reply src/pds/registrysweepers/utils/db/__init__.py]: resolved

File: src/pds/registrysweepers/utils/db/__init__.py (hunk 1 above)
Comment (alexdunnjpl) [reply src/pds/registrysweepers/utils/db/__init__.py]: resolved

**Review by nutjob4life — APPROVED** [review]

**Pull request merged**